core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -99,7 +99,7 @@ package object config {
       .createWithDefault(false)
 
   private[spark] val METRICS_PROCESS_TREE_METRICS =
-    ConfigBuilder("spark.metrics.logStageExecutorProcessTreeMetrics.enabled")
+    ConfigBuilder("spark.metrics.AddProcessTreeMetricsToMetricsSet.enabled")
```

Comment:
nit: "Add" should be changed to "add" to follow naming conventions

Reply:
thanks for catching this.